src/components/templates/Navigation.jsx

```diff
@@ -2,10 +2,15 @@
 import { createVNode } from "../../lib";
 import { router } from "../../router";
 import { globalStore } from "../../stores";
+import { BASE_PATH } from "../../constants/constants";
 
 const getNavItemClass = (path) => {
-  const currentPath = window.location.pathname;
-  return currentPath === path ? "text-blue-600 font-bold" : "text-gray-600";
+  let currentPath = window.location.pathname;
```

Comment:
이 부분은 currentPath가 이후에 재할당되지 않는다면 const로 선언해도 좋을 것 같아용~!


```suggestion
  const currentPath = window.location.pathname;
```